lhotse/bin/modes/recipes/__init__.py

```diff
@@ -59,3 +59,4 @@
 from .voxceleb import *
 from .wenet_speech import *
 from .yesno import *
+from .xbmu_amdo31 import *
```

Comment:
please also update
https://github.com/lhotse-speech/lhotse/blob/master/docs/corpus.rst

Reply:
Also, please sort the import alphabetically, i.e.,
```python3
from .xbmu_amdo31 import *
from .yesno import *
```

Reply:
OK